src/fontra/views/fontoverview/fontoverview.js

```diff
@@ -340,6 +346,16 @@ export class FontOverviewController extends ViewController {
     const glyphItemList = this.glyphOrganizer.filterGlyphs(combinedGlyphItemList);
     const glyphSections = this.glyphOrganizer.groupGlyphs(glyphItemList);
     this.glyphCellView.setGlyphSections(glyphSections);
+
+    // Show placeholder if no glyphs are found
+    const fontOverviewNoGlyphsContainer = document.querySelector(
+      "#font-overview-no-glyphs"
+    );
+    if (glyphSections.length === 0) {
```

Comment:
Simpler:
```js
    fontOverviewNoGlyphsContainer.classList.toggle("hidden", !glyphSections.length);
```

Reply:
Thanks. I changed it, but `!glyphSections.length` was wrong -> without `!` is correct.